docs/document/i18n/ja/docusaurus-plugin-content-docs/current/guides/in-office-day-list-customization.md

```diff
@@ -0,0 +1,316 @@
+  });
+  input.addEventListener("blur", () => {
+    input.style.border = "1px solid #e3e7e8";
+  });
+  return input;
+};
+
+const renderDay = (cellData) => {
+  return new Kuc.MultiChoice({
+    items: [
+      {
+        label: "Monday",
+        value: "1",
+      },
+      {
+        label: "Tuesday",
+        value: "2",
+      },
+      {
+        label: "Wednesday",
+        value: "3",
+      },
+      {
+        label: "Thursday",
+        value: "4",
+      },
+      {
+        label: "Friday",
+        value: "5",
+      },
+    ],
+    value: cellData,
+  });
+};
+
+const renderLocation = (cellData) => {
+  return new Kuc.Dropdown({
+    items: [
+      {
+        label: "-----",
+        value: "-",
+      },
+      {
+        label: "Tokyo",
+        value: "Tokyo",
+      },
+      {
+        label: "Vietnam",
+        value: "Vietnam",
+      },
+      {
+        label: "Shanghai",
+        value: "Shanghai",
+      },
+    ],
+    value: cellData,
+  });
+};
+
+const renderNote = (cellData) => {
+  return new Kuc.Text({
+    value: cellData,
+  });
+};
+
+const renderAction = (cellData) => {
+  const button = document.createElement("button");
+  const tooltipForButton = new Kuc.Tooltip({
+    title: "Submit changes or additions for this entry.",
+    container: button,
+    describeChild: true,
+    placement: "bottom",
+  });
+  return tooltipForButton;
+};
+
+const kucTeamTable = new Kuc.Table({
+  label: "team info",
+  columns: [
+    {
+      title: "Name",
+      field: "name",
+      render: renderName,
+    },
+    {
+      title: "In office day",
+      field: "day",
+      render: renderDay,
+    },
+    {
+      title: "Location",
+      field: "location",
+      render: renderLocation,
+    },
+    {
+      title: "Note",
+      field: "note",
+      render: renderNote,
+    },
+    {
+      title: "Action",
+      field: "action",
+      render: renderAction,
+    },
+  ],
+  data: [
+    {
+      name: "",
+      day: [],
+      location: "-",
+      note: "",
+    },
+  ],
+});
+```
+
+#### FieldGroup に Table コンポーネントを追加
+
+"KUC team" というラベルの FieldGroup を作成し、Table コンポーネントを追加します。
+
+```javascript
+const kucTeamFieldGroup = new Kuc.FieldGroup({
+  label: "KUC team",
+  content: kucTeamTable,
+});
+```
+
+#### FieldGroup に Tooltip を追加
+
+FieldGroup に Tooltip を追加します。
+
+```javascript
+const kucTeamTooltip = new Kuc.Tooltip({
+  title: "Office day info",
+  container: kucTeamFieldGroup,
+});
+```
+
+#### FieldGroup にイベントリスナーを追加
+
+FieldGroup の開閉で Tooltip のメッセージを変えるため、FieldGroup にイベントリスナーを追加します。
+
+```javascript
+kucTeamFieldGroup.addEventListener("change", (e) => {
+  if (e.detail.expanded) {
+    kucTeamFieldGroup.title = "";
+  } else {
+    kucTeamFieldGroup.title = "Office day info";
+  }
+});
+```
+
+#### UI 上の表示
+
+![render](/img/tooltip_fieldgroup_customize.gif)
+
+### Tooltip の２種類のユースケース
+
+ここでは [`describeChild`](../components/desktop/tooltip.md#property) というプロパティーを紹介します。
```

Comment:
No need of `` for Japanese, because users can distinguish property name and normal sentence with JA and EN.
```suggestion
ここでは [describeChild](../components/desktop/tooltip.md#property) というプロパティーを紹介します。
```